Cython/Debugger/DebugWriter.py

```diff
@@ -20,6 +20,13 @@
 from ..Compiler import Errors
 
 
+def is_valid_tag(name):
+    if hasattr(name, "startswith"):
+        if name.startswith(".") and name[1:].isnumeric():
```

Comment:
1. Probably worth an explanatory comment here - something like "names like `.0` are used internally for arguments to functions creating generator expressions, however they are not identifiers"
2. I suspect a type-check would be better than `hasattr`

Reply:
Addressed